.devcontainer/Dockerfile

```diff
@@ -13,7 +13,10 @@ RUN sudo apt-get install -y python3 python3-pip
 RUN pip install aws-cdk-lib==2.48.0 
 RUN pip install "constructs>=10.0.0,<11.0.0"
 RUN pip install boto3
+RUN pip install requests
 RUN pip install opensearch-py
+RUN pip install python-jose
+RUN pip install aws_cdk.aws_lambda_python_alpha
```

Comment:
```suggestion
```

We should let the **package** define the dependencies rather than through the dockerfile so we are all kept in-sync even if we aren't using Docker. So you'd add these dependencies to `pyproject.toml`. Then `pip install -e .[dev]` to get the dependencies installed into this environment. These should also be the dependencies required to run the CDK/infrastructure and not the lambda's as well.

Reply:
I had these in here primarily because Codespaces doesn't have access to the repo while its building the docker image, but I did find out a way to automatically run a pip install command after container is built.  So that's what it'll do now.